perun-openapi/openapi.yml

```
@@ -9101,6 +9101,22 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/membersManager/getRichMembersWithoutUserAttributes:
```

```suggestion
  /json/membersManager/getRichMembersNoUserAttributes:
```
This has to match the method name in MembersManager.